src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -241,10 +243,9 @@ OperatorSet OperatorsBridge::get_operator_set(const std::string& domain, int64_t
     }
     for (const auto& op : dm->second) {
         const auto& it = find(version, op.second);
-        if (it == std::end(op.second)) {
-            throw error::UnsupportedVersion{op.first, version, domain};
```

Comment:
why is this removed?

Reply:
I had a second look on this and actually I should have register Trilu to opset 1. So I can revert what I have removed.

The long story is that if I register Trilu to opset 14, the check in this loop breaks models that have opset version less than 14.
This loop scans every registered operator in ONNX frontend and if the operator version is greater than the version from the model that's being read - it throws this exception. For example, reading model with just Add node and opset version 7 will trigger this exception if any there's at least one operator in ONNX frontend that doesn't have version <= 7.